src/rdiff_backup/Globals.py

```diff
@@ -20,10 +20,14 @@
 
 import re
 import os
-from . import Version
 
 # The current version of rdiff-backup
-version = Version.version
+# Get it from package info or fall back to DEV version.
+try:
+    import pkg_resources
+    version = pkg_resources.get_distribution("rdiffweb").version
```

Comment:
You probably mean rdiff_backup instead of rdiffweb here.

Reply:
Yes, now you know where I copy-paste it from ! :)